apps/admin-gui/src/app/shared/components/dialogs/delete-entity-dialog/delete-entity-dialog.component.html

```diff
@@ -53,6 +53,9 @@ <h1 mat-dialog-title>{{'DIALOGS.DELETE_ENTITY.DELETE' | translate}}
           </span>
         </mat-list-item>
       </mat-list>
+      <div *ngIf="showAnotherMessage">
+        {{anotherMessage}}
+      </div>
```

Comment:
```suggestion
      <div *ngIf="anotherMessage">
        {{anotherMessage}}
      </div>
```

i think that showAnotherMessage = bool; completely useless and we dont need it

Reply:
Fixed